fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -434,28 +432,65 @@ public PlanFragment visitPhysicalProject(PhysicalProject<Plan> project, PlanTran
                 .map(e -> ExpressionTranslator.translate(e, context))
                 .collect(Collectors.toList());
         // TODO: fix the project alias of an aliased relation.
+        List<Slot> slotList = project.getOutput();
+        TupleDescriptor tupleDescriptor = generateTupleDesc(slotList, null, context);
         PlanNode inputPlanNode = inputFragment.getPlanRoot();
+        if (inputPlanNode instanceof HashJoinNode) {
```

Comment:
add a comment to explain why HashJoinNode is special and add a TODO that remove it when BE fix this special case

Reply:
done